web/views/layout.erb

```diff
@@ -29,5 +28,10 @@
     </div>
     <%= erb :_footer %>
     <%= erb :_poll_js %>
+    <script type="text/javascript">
```

Comment:
Also, it's worth noting that [`_poll_js`](https://github.com/mperham/sidekiq/blob/master/web/views/_poll_js.erb) already uses inline JavaScript.